lib/datapacksjob.js

```diff
@@ -2961,12 +2963,69 @@ DataPacksJob.prototype.downloadPerformanceData = async function (jobInfo) {
                 records.push(record);
             })
             .on("end", async function () {
-                var res = records.map(item => {
-                    return item[field_name];
+               
+                var omni_id =records.map(item => {
+                    return JSON.parse(item[field_name]).OmniScriptId;
                 });
 
-                VlocityUtils.log(`${i} of ${totalEntries.totalSize}`);
+                var unique_omni_id=Array.from([...new Set(omni_id)]);
+ 
+                var unique_omni_id_format="";
+                
+                for(var j=0;j<unique_omni_id.length;j++){
```

Comment:
please use spaces between variables and operators like:
for(var j=0; j < unique_omni_id.length; j++) {

Reply:
spaces provided between variables and operators